site/gatsby-site/src/components/RelatedIncidents.js

```diff
@@ -118,7 +118,7 @@ const allSearchColumns = {
       return (
         <>
           <Trans authors={authors}>
-            Incidents reports matched by authors: <b>{{ authors }}</b>
+            Incidents reports matched by authors: <b className="break-all">{{ authors }}</b>
```

Comment:
This change forces breaks a
t
e
v
e
r
y
l
e
t
t
e
r

doesn't seem like we want this 🤔